tests/test_doc_gate.py

```
@@ -904,3 +904,236 @@ def test_empty_trailer_commit_fails(self, tmp_path: Path):
+        repo.mkdir()
+        wf_dir = repo / ".github" / "workflows"
+        wf_dir.mkdir(parents=True)
+        (wf_dir / "ci.yml").write_text(
+            "name: CI\non: push\njobs:\n  build:\n    runs-on: ubuntu-latest\n"
+            "    steps:\n      - uses: actions/checkout@v4\n"
+        )
+        docs_dir = repo / "docs"
+        docs_dir.mkdir()
+        config_path = docs_dir / "doc-gate.toml"
+        config_path.write_text(self.CONTRIBUTOR_CONFIG)
+        subprocess.run(["git", "init"], cwd=repo, check=True, capture_output=True)
```

Ruff (0.16.2)

[error] Starting a process with a partial executable path

(S607)